package.json

```diff
@@ -203,9 +203,10 @@
     "hypercore-fetch": "^9.9.0",
     "ipfs-http-client": "^60.0.0",
     "ipfsd-ctl": "^13.0.0",
-    "js-ipfs-fetch": "^5.0.0",
+    "js-ipfs-fetch": "^3.0.1",
     "mime": "^2.4.6",
     "multiformats": "^9.9.0",
+    "node": "^21.2.0",
```

Comment:
Are the changes to the package json necessary? Mind reverting them (and the yarn.lock) and seeing if stuff still works?